docs/platforms/go/common/integrations/index.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Integrations
+description: "Learn about the automatic integrations Sentry provides and how to configure them."
+sidebar_order: 500
+---
+
+## Default Integrations
+
+These integrations are enabled by default and integrate into the standard library or the Go runtime itself. They are documented so you can be aware of what they do and disable them if they cause issues.
+
+To disable default integrations, you can provide an empty list of integrations when calling `sentry.Init()`.
+
+```go
+sentry.Init(sentry.ClientOptions{
+    Dsn:               "___PUBLIC_DSN___",
+	Integrations: func(i []Integration) []Integration {
+		return []Integration{}
+	    },
+    }
+)
+```
```

Comment:
Indentation is off.

Reply:
Same here:

![image](https://github.com/user-attachments/assets/b3edb525-1383-43ef-9767-8bb53c7f9d7b)